src/components/RegularBtn.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import { StyleSheet, TouchableOpacity, Text } from 'react-native';
+
+
+const RegularBtn = ( {navigation}: any ) => {
```

Comment:
You don't have another button, right? Would it make sense to call this component just `Button`?

Reply:
And pass title to it?